corehq/apps/app_manager/models.py

```diff
@@ -2032,12 +2032,21 @@ def has_persistent_tile(self):
         Return True if configured to persist a case tile on forms
         """
         return self.persist_tile_on_forms and (self.use_case_tiles or self.custom_xml)
-    
-    def set_other_configuration(self, src_module_detail_type):
-        for a in src_module_detail_type._obj:
-            if a.startswith('__') or a.startswith('_') or a in ['columns', 'filter']:
-                continue
-            setattr(self, a, getattr(src_module_detail_type, a))
+
+    def overwrite_from_module_detail(self, src_module_detail_type, attr_dict):
+        src_module_attrs = src_module_detail_type._obj.copy()
```

Comment:
It looks like you're just using this as a list of keys, can you use `keys` instead of `copy`?